app/src/main/res/layout/sync_tabs_error_row.xml

```diff
@@ -24,12 +24,12 @@
 
     <com.google.android.material.button.MaterialButton
         android:id="@+id/sync_tabs_error_cta_button"
-        android:layout_width="wrap_content"
+        style="@style/PositiveButton"
+        android:layout_width="341dp"
```

Comment:
Why 341dp? Should this maybe be `match_parent` like the parent `LinearLayout`? 

Thanks for the patch! 🙂

Reply:
@jonalmeida  so should i change it again ?